src/TrueLayer/ApiClient.cs

```diff
@@ -140,11 +126,6 @@ private async Task<ApiResponse<TData>> CreateResponseAsync<TData>(HttpResponseMe
             httpResponse.Headers.TryGetValues(CustomHeaders.TraceId, out var traceIdHeader);
             string? traceId = traceIdHeader?.FirstOrDefault();
 
-            if (httpResponse.StatusCode == HttpStatusCode.Unauthorized)
-            {
```

Comment:
Was this moved elsewhere?

Reply:
this is one of the corners that have been cut.
it's a bit of problem because it will require also the Api client to be keyed and it should also need to know which of the keyed cache to clear.

Reply:
it has been reintroduced in the last commit